src/App.js

```diff
@@ -13,11 +13,20 @@ function App() {
   return(
     <div>
       <input
-        type="search"
+        type='search'
         onInput={handleInput}
+        className='search-bar'
+        placeholder='Search wiki'
       />
-      {articles.map((article,i) => (
-        <li key={i}>{article}</li>
+      {articles.map((article) => (
+        Array.isArray(article)
+          ? article.map(item =>
+            item.length && item.includes('http')
```

Comment:
This check seems really obscure, can you add a comment about what's going on here?

Reply:
This is for having only links in search results

Reply:
Yeah a comment here to explain this would make it much better